sigs/contributor-strategy.md

```diff
@@ -0,0 +1,173 @@
+prospective new projects by developing community graduation criteria check
+points for rolling feedback and guidance.
+* Educate and Engage. Providing guidance to end users on how to engage with
+ contributors and vice versa.
+
+#### In scope:
+The following, non exhaustive, bootstrap list of activities and deliverables are
+in-scope for the SIG:
+* Definition of a contributor. This is helpful across projects for metrics and
+establishing guidelines, programs, and workflows.
+* “Contributing health checks”/’community health checks’ (name tbd) for project
+evaluations at graduation time.
+* Webinars, meetings, and other events to engage with the end user community on
+upstream contributing trainings and engagement programs.  
+* Development of guidelines, documents for project governance, recruiting and
+retaining contributor communities, mentorship, and project maturity.
+* Collection of current state of contributor strategies and governance models
+via surveys, GB reps, and Maintainers Circle (Example: what is the project doing
+  now, challenges, gaps)
+
+#### Out of scope
+* The day to day operations of CNCF SIGs, Kubernetes SIGs, or any community group
+of CNCF or its respective projects of any graduation level.
+* The creation and approval of CNCF SIGs or other community groups; we will offer advice but the responsibility lies on the TOC for those matters.
+* CNCF operations and marketing initiatives such as: product review/demo
+webinars, kubecon event planning, branding, stickers, swag, etc
+* Licensing and legal matters
+* Testing
+
+
+## Roadmap
+###1) SIG Formation:
+Role creation
+Stakeholder reps recruited and identified  
+
+###2) Discovery:
+with CNCF SIGs, projects, and end user community for contributor best practices,
+gaps/needs, current operations/programs, what hasn’t worked, etc.  
+
+###3) Establish working groups
+Create them from discovery or already known gaps while #2 is ongoing. Examples
+include:  
+1. Maintainers Circle
+2. Contributor growth and outreach: docs, diversity, recruitment, retention
+ * includes modern mentoring, succession planning, and staffing contributor role strategies
+3. "Community/Contributor health check"
+  * evaluation criteria  
+  * check-in/review/consulting process  
+4. Open governance guidelines and governance operations best practices
+  * the why, how, and where, your contributors make decisions  
+  * contributor diversity
+
+*Possible future roadmap projects*  
+If you see something here that interests you, join us and start it:  
+* Training: leadership, code of conduct, code reviewing, etc  
+* Contributor metrics and definitions  
+* Automation and self service for contributors, community GitOps
+
+## Governance
+This SIGs topic requires cross collaboration between end users, CNCF SIGs, and
+CNCF projects of all graduation levels.
+
+This SIG should be populated and governed by reps from CNCF projects that want
```

Comment:
Are you saying that if someone is not representing a CNCF project that you would not want them in a leadership role?  This might exclude experts in this area that may be interested in helping but are new to the CNCF, which is ok, just want to be clear.  You could always steer those folks to volunteer with a project first, but could lose some independent cross-project / ecosystem visibility.

Reply:
You know, that was a possibility that we didn't discuss at all, because it didn't occur to us that anyone who wasn't already involved with cncf projects would be interested.  We'll have to hash that out at the next meeting.

Reply:
I think from a scope and TOC perspective, we may still want to focus on CNCF projects from the scope of SIG Contributor Strategy. The TOC's mission is still oriented around those projects, and the SIG is a body of the TOC. I'd defer to the TOC on that scope, but while we shouldn't be kingmakers, we should still assist the TOC in their mission -- and SIG contributor strategy in that part is about projects that are part of the CNCF and aiding the TOC and GB in supporting them.

Reply:
Although, I do agree @ultrasaurus that leadership shouldn't be dependent on existing CNCF project leadership.

Reply:
I just want to make sure that this SIG is focused on empowering CNCF projects, as is from what I understood of the charter. Not to be kingmakers - but part of this SIG would be to help projects and maintainers on their journey from (this may be different depending on project):

* enter the CNCF at the sandbox stage
* move to incubating
* move to graduated

and have resources along the way. while I think anyone should be able to join, we need to keep our focus clear.

Reply:
I added a general sentiment around welcoming industry experts; ptal. i'm mostly open to the idea if they have relevant experience, especially building contributor bases and governance for other oss projects that we can learn from. i think bootstrap leadership may make sense with reps from cncf sigs/cncf projects only to assess and kick things off.

Reply:
I agree with @gerred - I prefer that the SIG members and leaders be made up of people active with CNCF projects. We can certainly have outside experts offer insight, etc as needed, but I feel that should be as a resource, not a member. It's similar to how our OSS projects function - all are welcome but a level of engagement/involvement is needed before taking a leadership role. I don't think being active with/in a CNCF project is a barrier here.

Reply:
:+1: